### ISSUE TITLE
mail theme should follow personal site theme settings

### ISSUE THREAD

**Issue by mcnesium** [tootsuite/mastodon#10053]
When the site theme in a user's profile is e.g. set to "Mastodon (light)", the (HTML-)mail notifications should be themed the same way to have a consistent user experience.

![mstdn1](https://user-images.githubusercontent.com/476060/52859448-09d8c000-312d-11e9-8285-5d9e9bb5bcf7.png) ![mstdn2](https://user-images.githubusercontent.com/476060/52859449-09d8c000-312d-11e9-963a-e9530cfc0d7e.png)

**Comment by mynameismonkey** [comment]
Agreed, and if styling is an issue, at a minimum send the mail in text only if theme = Mastodon (light). As admin of a custom theme instance, these E-mails are quite jarring for the newer members, they have never participated with a dark "Mastodon" - instead they were expecting a light themed "toot.wales" message. We have a relatively less techy audience.
